trade-finance-manager-ui/templates/case/deal/deal.njk

```diff
@@ -92,10 +92,6 @@
         </div>
       </div>
       <div class="govuk-grid-row govuk-!-margin-0 govuk-!-padding-top-6">
-        {# <div class="govuk-grid-column-one-fifth govuk-!-padding-bottom-4 govuk-!-padding-left-0">
```

Comment:
Looks good mate, could you please write a one liner on why it is commented. :-)

Reply:
actually, this was removed already :)
so there is no need for a comment